openslides_backend/action/speaker/create_update_delete.py

```diff
@@ -17,3 +24,8 @@ class SpeakerActionSet(ActionSet):
     )
     update_schema = DefaultSchema(Speaker()).get_update_schema(["marked"])
     delete_schema = DefaultSchema(Speaker()).get_delete_schema()
+    routes = {
```

Comment:
The syntax here has changed, see https://github.com/OpenSlides/openslides-backend/pull/205

Reply:
Done.